tests/integration/data_sources_and_expectations/expectations/test_expect_compound_column_values_to_be_unique.py

```diff
@@ -28,7 +40,10 @@
 
 @parameterize_batch_for_data_sources(data_source_configs=ALL_DATA_SOURCES, data=DATA)
 def test_golden_path(batch_for_datasource: Batch) -> None:
-    expectation = gxe.ExpectCompoundColumnsToBeUnique(column_list=[STRING_COL, INT_COL, INT_COL_2])
+    expectation = gxe.ExpectCompoundColumnsToBeUnique(
+        column_list=[STRING_COL, INT_COL, INT_COL_2],
+        ignore_row_if="any_value_is_missing",
```

Comment:
This test should not have been passing without this flag set. It seems like there is still a bug that allows it to pass with SQL and pandas from my understanding. The sets [`None`, `None`, `None`] are not unique.